lib/src/rules/cascade_invocations.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.rules.cascade_invocations;
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/linter.dart';
+
+const _desc = r'Cascade consecutive method invocations on the same reference.';
+
+const _details = r'''
+
+**DO** Use the cascading style when succesively invoking methods on the same
+ reference.
```

Comment:
Perhaps update the documentation to include getters? Also, do we care about setters:
```
x.m();
x.s = '';
x.m();
```